independent-publisher-connectors/NFTMania/NFTMania.md

```diff
@@ -0,0 +1,38 @@
+**Proposal for NFTMania Connector**
```

Comment:
Please use this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md) for the readme file. you need to use the "#" key. you can see the raw code [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/Independent%20Publisher/readme.md)

Please include the required sections.